lib/requireContent.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+  var global = this;
+  require.load = function (context, moduleName, url) {
+    var xhr;
+    xhr = new XMLHttpRequest();
+    xhr.open("GET", chrome.extension.getURL(url) + '?r=' + new Date().getTime(), true);
+    xhr.onreadystatechange = function (e) {
+      if (xhr.readyState === 4 && xhr.status === 200) {
+        eval.call(global, xhr.responseText + '\n//@ sourceURL=' + url);
+        context.completeLoad(moduleName)
+      }
+    };
+    xhr.send(null);
+  };
+})();
```

Comment:
This is weird. Why is this?

The `?r=date` part can be done just in require config I'd expect (set [urlArgs](http://requirejs.org/docs/api.html#config-urlArgs) to "r=" + Date.now()).

It looks like you're changing how require.load works to use AJAX instead of it's built-in mechanisms. That will work fine but it's pretty weird and should be unnecessary; normal loading should work as-is, even in a Chrome extension. Happy to stick with it if you've got a good reason, but it'd be nice to document that reason here.